### ISSUE TITLE
Training Details about diffused heads

### ISSUE THREAD

**Issue by xiapengchng** [MStypulkowski/diffused-heads#10]
Thank you for your awsome work!
Could you tell me the training details about diffuse heads such as learning rate, batch size, optimizer  since i want to reimplement the training pipeline.

**Comment by MStypulkowski** [comment]
Thank you and sorry for late reply!

Learning rate: 5e-5 decreasing below 1e-6 with 0.8 factor.
Batch size: 10 per GPU. 
Optimizer: Adam.
